authui/src/authgear.ts

```diff
@@ -265,6 +265,14 @@ window.api.onLoad(() => {
   };
 });
 
+// Handle auto form submission
+window.api.onLoad(() => {
+  const e = document.querySelector('[data-auto-submit="true"]');
```

Comment:
When I press back button after it auto-submitted, it shows `There is a problem with the network.` error since it attempts to auto-submit again. I suppose we should auto-submit at most once per page?

Reply:
I made the auto-submit runs at most once